fixtures/dom/src/components/fixtures/input-change-events/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import FixtureSet from '../../FixtureSet';
+import TestCase from '../../TestCase';
+import RangeKeyboardFixture from './RangeKeyboardFixture';
+import RadioClickFixture from './RadioClickFixture';
+import InputPlaceholderFixture from './InputPlaceholderFixture';
+
+class InputChangeEvents extends React.Component {
+  render() {
+    return (
+      <FixtureSet
+        title="Input change events"
+        description="Tests proper behavior of the onChange event for inputs"
+      >
+        <TestCase
+          title="Range keyboard changes"
+          description={`
+            Range inputs should fire onChange events for keyboard events
+          `}
+        >
+          <TestCase.Steps>
+            <li>Focus range input</li>
+            <li>change value via the keyboard arrow keys</li>
+          </TestCase.Steps>
+
+          <TestCase.ExpectedResult>
+            The <code>onKeyDown</code> call count should be equal to
+            the <code>onChange</code> call count.
+          </TestCase.ExpectedResult>
+
+          <RangeKeyboardFixture />
+        </TestCase>
+
+        <TestCase
+          title="Radio input clicks"
+          description={`
+            Radio inputs should only fire change events when the checked
+            state changes.
+          `}
+          minimumVersion="16.0.0"
```

Comment:
Maybe it would be helpful to also add `resolvedIn` and `resolvedBy` so we could give more details

```jsx
<TestCase title="foo" resolvedIn="15.2.0" resolvedBy="#1760" />
```